site/src/docs/components/footer/customisation.mdx

```diff
@@ -98,4 +98,4 @@ import { Footer, Logo, logoFi } from 'hds-react';
 )
 ```
 
-</Playground>
\ No newline at end of file
+</Playground>
```

Comment:
> Should the same changes be in the docs site too? Doc site doesn't now have the fancy dropdown language menu.

@mrTuomoK For now I'd say no, as the header docs have a notification in "With language selector" section that the additional languages functionality is coming. So maybe put this in docs when the logic works for switching?

Reply:
Welp, I just pushed changes to doc site 😄 https://github.com/City-of-Helsinki/helsinki-design-system/pull/1116/commits/cee056303627976d46cdcf0c26ff441bdcec8d8c

Reply:
Less in more, I can revert them. 🤔

Reply:
I'm slightly confused what we have and where, as these things are happening all over the place

But, what I want for release is: 
- working examples: if projects copy the examples they are not broken. Examples can be subset of features, not all examples need everything. If example has language menu, it should change language, otherwise it does not provide value for user.
- documentation matches what we have
- implemented features match design
- not all features need to be in 3.00, if features are not done they can wait for next release as long as it doesn't prevent projects using the component in the first place